### ISSUE TITLE
A font with full set of unicode emoji and pictographs

### ISSUE THREAD

**Issue by JamesCoyle** [Templarian/MaterialDesign#1133]
It would be nice to have a usable font which includes material design versions of the emoji and pictographs included in the unicode standard. There are already a few suitable icons in the MDI project which could be repackaged into a new usable font and all new icons can be included in both mdi and the emojifont.

**Comment by Templarian** [comment]
That could be a good use of the new subfont support that I've coded in.

Use the 2nd sheet tab to start this work (the backend is already coded for subfont, so it could be used to do this already).

https://docs.google.com/spreadsheets/d/1a-ZW7Fm_pKSfwkG1kxJ9Cszz_EI3g7GuYq1KaUZQag0/edit#gid=2087014927

**Comment by JamesCoyle** [comment]
I've made a start on it. Added the emoticon range and transport range so icons can just be dropped in when completed. Also put in the poop, happy. and sad values. Will need to decide what ranges to include other than those though. I suspect the entirety of [misc symbols and pictographs](http://unicode-table.com/en/#miscellaneous-symbols-and-pictographs) will also be included but I can't really be bothered to add that entire range in at the moment so will add it in later.
